### ISSUE TITLE
tempfile support

### ISSUE THREAD

**Issue by foresto** [Tinche/aiofiles#20]
I would have use for aiofiles if it supported creating and managing temporary files, especially via the standard [tempfile](https://docs.python.org/3/library/tempfile.html) interface. Potential new feature?

**Comment by Tinche** [comment]
Yeah I think this would be nice. If you have the time and energy go for it. I'd say add a `aiofiles.tempfile` module with coroutines mimicking the `tempfile` module API.
